app/assets/javascripts/app/mix_panel.js

```diff
@@ -45,7 +44,7 @@ App.addChild('MixPanel', {
     this.trackPageVisit('projects', 'new', 'Visited new project page');
     this.trackPageVisit('projects', 'index', 'Visited home');
     this.trackPageVisit('explore', 'index', 'Explored projects');
-    this.trackPageLoad('contributions', 'edit', 'Selected reward');
```

Comment:
```trackContributions``` is too complex.
I believe it's better to leave this line and use the trackContributions only for the contribution/new in order to make it simpler.